azext_iot/tests/digitaltwins/test_dt_import_jobs_int.py

```diff
@@ -118,12 +118,57 @@ def test_dt_importjobs(self):
         )
         valid_import_job_output_filename = create_valid_import_job_output["outputBlobUri"].split("/")[-1]
 
+        # Run through import job lifecycle
+        # Cancel import job (there could be a race condition)
+        self.cmd(
+            "dt job import cancel -n '{}' -g '{}' -j '{}' -y".format(instance_name, self.rg, valid_import_job_id)
+        )
+
+        # Poll to make sure job is cancelled
+        num_tries = 0
+        while num_tries < MAX_TRIES:
+            num_tries += 1
+            show_import_job_output = self.cmd(
+                "dt job import show -n '{}' -g '{}' -j '{}'".format(instance_name, self.rg, valid_import_job_id)
+            ).get_output_in_json()
+            assert show_import_job_output["status"] in ["cancelled", "cancelling", "running"]
+            if show_import_job_output["status"] == "cancelled":
+                assert show_import_job_output["error"] is None
+                break
+            sleep(POLL_SLEEP_INTERVAL)
+
+        # Delete import job
+        self.cmd(
+            "dt job import delete -n '{}' -g '{}' -j '{}' -y".format(instance_name, self.rg, valid_import_job_id)
+        )
+
+        # Recreate import job
+        self.cmd(
+            "dt job import create -n '{}' -g '{}' -j '{}' --df '{}' --ibc '{}' --isa '{}'".format(
+                instance_name, self.rg, valid_import_job_id,
+                valid_import_data_filename, self.storage_container, self.storage_account_name
+            )
+        ).get_output_in_json()
+
         # Show import job
         show_import_job_output = self.cmd(
             "dt job import show -n '{}' -g '{}' -j '{}'".format(instance_name, self.rg, valid_import_job_id)
         ).get_output_in_json()
         assert show_import_job_output["id"] == valid_import_job_id
 
+        # Poll to ensure desired status of valid import job before starting new one
```

Comment:
This could be a test utility function "ensure_import_job_status(...)" or similar, may be code reuse opportunity across the Poll job status steps.